components/clarinet-cli/src/devnet/package.rs

```diff
@@ -0,0 +1,91 @@
+use std::fs::{read_to_string, File};
+use std::io::{self, Write};
+use std::process;
+
+use clarinet_deployments::types::DeploymentSpecificationFile;
+use clarinet_files::{FileLocation, NetworkManifestFile, ProjectManifestFile};
+
+#[derive(Serialize, Deserialize, Debug)]
+struct ConfigurationPackage {
+    deployment_plan: DeploymentSpecificationFile,
+    devnet_config: NetworkManifestFile,
+    clarinet_config: ProjectManifestFile,
+}
+
+fn get_devnet_config() -> Result<NetworkManifestFile, io::Error> {
+    let file_content = read_to_string("./settings/Devnet.toml")?;
```

Comment:
The `NetworkManifest`  has a `from_project_manifest_location` function, can we use that instead of a hard-coded path?

Reply:
Done.